shivatmica/solutions/04_median_of_two_sorted_arrays.py

```diff
@@ -1,25 +1,26 @@
 class Problem4:
```

Comment:
Mo asked me in Issue #12 to move all files "into a single, consolidated folder (not split into implementation and tests)." Can you please do the same for your files?

Reply:
Hi @Adarsh321123, I'd actually seen that issue and changed it accordingly, but when Mo reviewed my code, he asked me to separate the source code and tests into different subdirs in Issue #16. I can ask Mo which is the preferred format.

Reply:
Got it @shivatmica. Since Mo asked you to separate the source code and tests, let's go with that.

Reply:
Alright, sounds good!